assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -78,25 +83,14 @@ function PagePreview({ index, gridRef, ...props }) {
   }));
   const page = pages[index];
   const { backgroundColor } = page;
-  const { width: thumbWidth, height: thumbHeight, isActive } = props;
+  const { width: thumbWidth, height: thumbHeight, tabIndex } = props;
   const width = thumbWidth - THUMB_FRAME_WIDTH;
   const height = thumbHeight - THUMB_FRAME_HEIGHT;
-  const [tabIndex, setTabIndex] = useState(isActive ? 0 : -1);
-
-  const onBlur = (e) => {
-    if (gridRef?.current?.contains(e.relatedTarget)) {
-      setTabIndex(-1);
-    }
-  };
-
-  const onFocus = () => {
-    setTabIndex(0);
-  };
 
   return (
     <UnitsProvider pageSize={{ width, height }}>
       <TransformProvider>
-        <Page tabIndex={tabIndex} onBlur={onBlur} onFocus={onFocus} {...props}>
+        <Page tabIndex={tabIndex} {...props}>
```

Comment:
Given that `tabIndex` is just passed as-is to the `<Page />`, I think you can completely remove any notion of it in this component and just let it pass through via `props`.

Reply:
You still need to rename [`ariaLabel` to `aria-label`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/canvas/carousel/index.js#L418) in the parent <Carousel /> component (yeah, [React is a bit inconsistent with these props](https://reactjs.org/docs/accessibility.html#wai-aria:~:text=Note%20that%20all%20aria%2D*%20HTML%20attributes,as%20they%20are%20in%20plain%20HTML%3A)) to make it pass correctly through.

Reply:
Sorry, I mixed up the two PR's and added some irrelevant comments here. I've removed them and am in the process of re-applying them to #4260, where they belong.